src/style.css

```diff
@@ -1,3 +1,73 @@
+@font-face {
+  font-family: Roboto;
+  src: url(font/Roboto-Regular.ttf);
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url(font/Roboto-Medium.ttf);
+  font-weight: 500;
```

Comment:
You have defined the 'Roboto' font-family multiple times for different weights, which is good, but you should also include a fallback font-family for each @font-face rule. For example, 'font-family: Roboto, Arial, sans-serif;'. This ensures that if the custom font fails to load for some reason, the browser can use an alternative font that is likely to be available.